src/main/java/com/example/SteDziPanki/repository/ContinentRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.example.SteDziPanki.repository;
+
+import com.example.SteDziPanki.model.Continent;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ContinentRepository extends JpaRepository <Continent, Long> {
+
+    List<Continent> findAll();
```

Comment:
Ta metoda jest wbudowana w interfejs JpaRepository, więc nie ma sensu jej powtarzać tutaj.

To samo tyczy się reszty repozytoriów